frame/offences/benchmarking/src/lib.rs

```diff
@@ -175,6 +175,32 @@ fn make_offenders<T: Trait>(num_offenders: u32, num_nominators: u32) -> Result<
 	Ok((id_tuples, offenders))
 }
 
+fn make_offenders_im_online<T: Trait>(num_offenders: u32, num_nominators: u32) -> Result<
```

Comment:
These two methods could be de-duplicated if the conversion functions were passed externally. I can totally survive with this benchmark-only duplication though :)

Reply:
Then let's ignore this for now 🙈